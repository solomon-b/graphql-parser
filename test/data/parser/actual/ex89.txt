[ DefinitionTypeSystem
    ( TypeSystemDefinitionDirective
        ( DirectiveDefinition
            { _ddDescription = Nothing
            , _ddName = "example"
            , _ddArguments = []
            , _ddLocations =
                [ ExecDirLoc EDLFIELD
                , ExecDirLoc EDLFRAGMENT_SPREAD
                , ExecDirLoc EDLINLINE_FRAGMENT
                ]
            }
        )
    )
]