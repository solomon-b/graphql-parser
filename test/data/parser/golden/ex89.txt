[ DefinitionTypeSystem
    ( TypeSystemDefinitionDirective
        ( DirectiveDefinition
            { _ddSpan = Span
                { _start = AlexSourcePos
                    { _line = 0
                    , _col = 1
                    }
                , _end = AlexSourcePos
                    { _line = 3
                    , _col = 20
                    }
                }
            , _ddDescription = Nothing
            , _ddName = "example"
            , _ddArguments = Nothing
            , _ddRepeatable = NotRepeatable
            , _ddLocations =
                [ ExecDirLoc
                    ( Span
                        { _start = AlexSourcePos
                            { _line = 1
                            , _col = 5
                            }
                        , _end = AlexSourcePos
                            { _line = 1
                            , _col = 10
                            }
                        }
                    ) EDLFIELD
                , ExecDirLoc
                    ( Span
                        { _start = AlexSourcePos
                            { _line = 2
                            , _col = 5
                            }
                        , _end = AlexSourcePos
                            { _line = 2
                            , _col = 20
                            }
                        }
                    ) EDLFRAGMENT_SPREAD
                , ExecDirLoc
                    ( Span
                        { _start = AlexSourcePos
                            { _line = 3
                            , _col = 5
                            }
                        , _end = AlexSourcePos
                            { _line = 3
                            , _col = 20
                            }
                        }
                    ) EDLINLINE_FRAGMENT
                ]
            }
        )
    )
]